dataView/DataView.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {Component} from 'react';
+import {hoistComponent, elemFactory} from 'hoist/core';
+import {grid} from 'hoist/grid';
+import {GridModel} from 'hoist/grid';
+import {baseCol} from 'hoist/columns/Core';
+
+@hoistComponent()
+class DataView extends Component {
+
+    constructor(props) {
+        super(props);
+        const {store, selection, contextMenuFn, itemFactory} = props.model;
+        this._gridModel = new GridModel({
+            store,
+            selection,
+            contextMenuFn,
+            columns: [
+                baseCol({
+                    field: 'id',
+                    flex: 1,
+                    autoHeight: true,
+                    renderElement: (props) => {
+                        return itemFactory({record: props.data});
```

Comment:
use of "props" here for the parameter name is a bit confusing.  This is really the grid record being rendered, not the props of the data view or constructor this code sits in.  

Also, think if we go ahead with this idea of a BaseCol enhancement, the name of the property should be something like  or 'elementRenderer' or 'elementCellRenderer'.   This is really analogous to a cellRenerder, which is an existing API object in AG grid.